packages/driver/src/dom/elements.ts

```diff
@@ -864,6 +864,11 @@ const getFirstFixedOrStickyPositionParent = ($el) => {
     return $el
   }
 
+  // the current element is in a shadowDom
+  if ($el.offsetParent().length > 0 && !$el.offsetParent()[0].contains($el[0])) {
```

Comment:
if this is really just to check if the element is in a shadow root, you should instead rely on methods the browser already has for this:

```
if ($el[0].getRootNode() instanceof ShadowRoot) {
```